e2e/cypress/plugins/smtpTester.js

```diff
@@ -0,0 +1,34 @@
+const ms = require('smtp-tester');
+
+const initialize = (on, config) => {
+  // starts the SMTP server at localhost:7777
+  const port = 7777;
+  const mailServer = ms.init(port);
+  console.log('mail server at port %d', port);
+
+  let lastEmail = {};
+
+  // process all emails
+  mailServer.bind((addr, id, email) => {
+    console.log('email: ', email);
+    lastEmail[email.headers.to] = email;
+  });
+
+  on('task', {
+    resetEmails(email) {
```

Comment:
what is this for?
should it be `resetEmail(toRecipient)` ?

Reply:
It's to delete the emails in case we have multiple tests and want to be sure we are looking at the right one. It's not used for now because I added only two tests but I think it can be useful for the future. I renamed the parameter as `recipient` 👍